smart-contracts/contracts/PublicLock.sol

```diff
@@ -119,19 +119,20 @@ contract PublicLock is ILockPublic {
     uint _keyPrice,
     uint _maxNumberOfKeys
   )
-    public {
-      unlockProtocol = msg.sender; // Make sure we link back to Unlock's smart contract. (TODO: handle upgrades?)
-      owner = _owner;
-      expirationDuration = _expirationDuration;
-      keyPrice = _keyPrice;
-      if (_maxNumberOfKeys == uint256(-1)) {
-        maxNumberOfKeys = uint256(-1);
-      } else {
-        maxNumberOfKeys = _maxNumberOfKeys;
-      }
-
+    public
+  {
+    unlockProtocol = msg.sender; // Make sure we link back to Unlock's smart contract. (TODO: handle upgrades?)
+    transferOwnership(_owner);
+    expirationDuration = _expirationDuration;
+    keyPrice = _keyPrice;
+    if (_maxNumberOfKeys == uint256(-1)) {
```

Comment:
Now that I read this, wouldn't me make sense to just have 
`maxNumberOfKeys = _maxNumberOfKeys;` since that it was it does in both cases?

Reply:
Yep, you're right. done.